test/e2e/serial/workload_placement_test.go

```diff
@@ -1054,19 +1177,45 @@ func makePaddingPod(namespace, nodeName string, zone nrtv1alpha1.Zone, podReqs c
 	return padPod, nil
 }
 
+func makeNodePaddingPod(namespace string, node corev1.Node, podReqs corev1.ResourceList) (*corev1.Pod, error) {
+	klog.Infof("want to have node %q with allocatable: %s", node.Name, e2ereslist.ToString(podReqs))
+
+	paddingReqs, err := e2enrt.SaturateNodeUntilLeft(node, podReqs)
+	if err != nil {
+		return nil, err
+	}
+
+	klog.Infof("padding resource to saturate %q: %s", node.Name, e2ereslist.ToString(paddingReqs))
+
+	padPod := objects.NewTestPodPause(namespace, fmt.Sprintf("padpod-%s-full", node.Name))
```

Comment:
turns out this way of creating pod names is wrong, can lead to spurious rejections for name too long. 
See: https://github.com/openshift-kni/numaresources-operator/commit/41460a9d5eee4c7a70631a65215a0248c0a40339
We can fix this and all the occurrences in a later PR though